packages/slappforge-sdk-redis/lib/Redis.d.ts

```diff
@@ -20,31 +20,199 @@
 export class Redis {
     constructor(clusterManager: ClusterManager);
 
-    set(prams: SETParams, callback: Function);
+    /**strings**/
+    append(params: APPENDParams, callback: Function);
 
-    get(prams: GETParams, callback: Function);
+    incrby(params: INCRBYParams, callback: Function);
 
-    del(prams: DELParams, callback: Function);
+    decrby(params: DECRBYParams, callback: Function);
 
+    get(params: GETParams, callback: Function);
+
+    set(params: SETParams, callback: Function);
+
+    setex(params: SETEXParams, callback: Function);
+
+    strlen(params: STRLENParams, callback: Function);
+
+    /**typeParams**/
+    del(params: DELParams, callback: Function);
+
+    expire(params: EXPIREParams, callback: Function);
+
+    persist(params: PERSISTParams, callback: Function);
+
+    type(params: TYPEParams, callback: Function);
+
+    rename(params: RENAMEParams, callback: Function);
+
+    /**sets**/
+    sadd(params: SADDParams, callback: Function);
+
+    scard(params: SCARDParams, callback: Function);
+
+    sdiff(params: SDIFFParams, callback: Function);
+
+    sdiffstore(params: SDIFFSTOREParams, callback: Function);
+
+    sinter(params: SINTERParams, callback: Function);
+
+    sinterstore(params: SINTERSTOREParams, callback: Function);
+
+    sismember(params: SISMEMBERParams, callback: Function);
+
+    smembers(params: SMEMBERSParams, callback: Function);
+
+    smove(params: SMOVEParams, callback: Function);
+
+    srem(params: SREMParams, callback: Function);
+
+    sunion(params: SUNIONParams, callback: Function);
+
+    sunionstore(params: SUNIONSTOREParams, callback: Function);
+}
+
+interface Key{
```

Comment:
Please keep a space between the interface name and the opening parenthesis. Applicable to next few interface definitions as well.